Assets/Scripts/Editor/InkDialogSystem.Editor.asmdef

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "InkDialogSystem.Editor",
```

Comment:
Same deal here where we should rename this to 'InkDialog**ue**System.Editor to keep consistent with the spelling of Dialogue.